### PR TITLE
Documentation on how to view dataspace catalog through the UI

### LINKED ISSUE #10: View Dataspace Catalog (Offers Shared with Me)
Feature #3 

## Description

As a dataspace participant I want to look at all the data assets that other participants offer to share with me.

The connector's web UI offers a page to display all data offerings, which can be obtained through the connector's REST API. 
The catalog crawler collects as `ContractOffer`s from all participants by obtaining the participant list from the `RegistrationService`. 

## Acceptance Criteria
- [x] Web UI lists all offerings. Advanced search/filtering is out of scope (will be in #87)
- [x] Offerings include data assets other participants offer to share with me.
- [x] Documentation on how to view offerings through the UI is available
- [x] Offerings can be viewed based on step by step instructions without having deep technical expertise

## Tasks

As the UI side is already implemented in the webapp, only a bug fix and documentation are outstanding.

- [ ]  <strike>Fix empty columns in UI (timestamp, Protocol)</strike>
- [x]  Documentation on how to view dataspace catalog through the UI is available

### LINKED ISSUE #131: View Dataspace Catalog (Offers Shared with Me)

### PR DESCRIPTION
**Pull request by cpeeyush**
Documentation on how to view dataspace catalog through the UI

Closes agera-edc/MinimumViableDataspaceFork#131

### CODE REVIEW

**Review by marcgs — APPROVED** [review]

**Comment by zeier** [comment]
Regarding the docs folder, it would make sense to have all the documentation for the EDC Data Dashboard in a dedicated folder e.g. `docs/developer/edc-data-dashboard`, since we will have multiple files and folders related to other topics in the `docs/developer` folder. 

I created a corresponding story https://github.com/agera-edc/MinimumViableDataspace/issues/133 for this, let's keep the docs where they are for now and move them later as part of this follow-up story.

File: docs/developer/view-catalog.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# View Catalog
+
+## Overview
+
+Dataspace participants can use EDC data dashboard to view what all contract offers are shared with them. It is called `Catalog` and it gets refreshed frequently as new contract offers are added by other participants.
```
Comment (zeier) [inline docs/developer/view-catalog.md]:
```suggestion
Dataspace participants can use the EDC data dashboard to view contract offers shared with them. Contract offers are added to the `Catalog`, which is refreshed frequently (the refresh interval is configurable per connector) as new contract offers are added by other participants.
```

**Comment by zeier** [comment]
Let's include a screenshot of the UI, which is makes it much easier to follow the instructions and comprehend the process, even if you have not yet deployed your MVD instance. This way, you can read through the docs once, and easily remember what to do when you fire up the data dashboard.

File: docs/developer/view-catalog.md (hunk 1 above)
Comment (chrislomonico) [inline docs/developer/view-catalog.md]: nit: 
```suggestion
Dataspace participants can use the EDC data dashboard to view  contract offers are shared with them. It is called `Catalog` and it gets refreshed frequently as new contract offers are added by other participants.
```

**Review by chrislomonico — APPROVED** [review]

File: docs/developer/view-catalog.md (hunk 1 above)
Comment (pkirch) [inline docs/developer/view-catalog.md]: Q for clarity: what gets refreshed the UI or the catalog? Or is it the same?

And what should the end user expect reading `frequently`? Is there a default value?

**Review by zeier — APPROVED** [review]
Let's include a screenshot of the UI, which is makes it much easier to follow the instructions and comprehend the process, even if you have not yet deployed your MVD instance. This way, you can read through the docs once, and easily remember what to do when you fire up the data dashboard.

Regarding the docs folder, it would make sense to have all the documentation for the EDC Data Dashboard in a dedicated folder e.g. docs/developer/edc-data-dashboard, since we will have multiple files and folders related to other topics in the docs/developer folder.

I created a corresponding story https://github.com/agera-edc/MinimumViableDataspace/issues/133 for this, let's keep the docs where they are for now and move them later as part of this follow-up story.

File: docs/developer/view-catalog.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Viewing Catalog/Offers
+
+* Go to `Dataspace Catalog` pane and here you will see all existing contract offers which are shared with you.
```
Comment (pkirch) [inline docs/developer/view-catalog.md]: Question on `you`: Is this sharing based on an end user identity or is it based on the `dataspace participant` (= EDC connector?)?

I'm trying to understand it in the context of the introduction:
`Dataspace participants can use EDC data dashboard to view what all contract offers are shared with them.`

**Review by pkirch — APPROVED** [review]
Minor questions for clarity.

No blockers.

**Comment by cpeeyush** [comment]
> Regarding the docs folder, it would make sense to have all the documentation for the EDC Data Dashboard in a dedicated folder e.g. `docs/developer/edc-data-dashboard`, since we will have multiple files and folders related to other topics in the `docs/developer` folder.
> 
> I created a corresponding story agera-edc/MinimumViableDataspaceFork#68 for this, let's keep the docs where they are for now and move them later as part of this follow-up story.

Moved to edc-data-dashboard folder as it was already created in other PR.

File: docs/developer/view-catalog.md (hunk 1 above)
Comment (cpeeyush) [reply docs/developer/view-catalog.md]: Applied.

Comment (cpeeyush) [reply docs/developer/view-catalog.md]: Applied.

Comment (cpeeyush) [reply docs/developer/view-catalog.md]: 1. Catalog gets refreshed in background and so the UI (not automatically, needs a page reload)
2. Added more information for further clarity. 

Thank you.

File: docs/developer/view-catalog.md (hunk 2 above)
Comment (cpeeyush) [reply docs/developer/view-catalog.md]: Update for more clarity. Thank you.

**Pull request merged**